### PR TITLE
Removed final bits of script blocks that were moved to the test module.

### PR DESCRIPTION
**Pull request by zionyx**
Cleaned up some duplicated script blocks used in tests.

### CODE REVIEW

File: Test-Source/Invoke-DockerPull.Tests.ps1 [hunk 1]
```diff
@@ -27,115 +19,90 @@ Describe 'Pull docker images' {
 
         It 'pulls public docker image by image name only' {
             Invoke-DockerPull -ImageName 'ubuntu'
-            $result = GetMockValue -Key "pull"
+            $result = GetMockValue -Key "command"
```
Comment (rasmusjelsgaard) [inline Test-Source/Invoke-DockerPull.Tests.ps1]: I suggest we also make the key name a constant (the name is long but at least it's descriptive):

```suggestion
            $result = GetMockValue -Key $Global:InvokeCommandReturnValueKeyName
```

File: Test-Source/Invoke-DockerPull.Tests.ps1 [hunk 2]
```diff
@@ -27,115 +19,90 @@ Describe 'Pull docker images' {
...
-                $commandResult = [CommandResult]::new()
-                $commandResult.ExitCode = 1
-                return $commandResult
-            }
-            Mock -CommandName "Invoke-Command" $returnExitCodeOne  -Verifiable -ModuleName $Global:ModuleName
-            $theCode = {
-                Invoke-DockerPull -ImageName 'mcr.microsoft.com/ubuntu'
-            }
+            Mock -CommandName "Invoke-Command" $Global:CodeThatReturnsExitCodeOne  -Verifiable -ModuleName $Global:ModuleName
+            $theCode = { Invoke-DockerPull -ImageName 'mcr.microsoft.com/ubuntu' }
```
Comment (rasmusjelsgaard) [inline Test-Source/Invoke-DockerPull.Tests.ps1]: Should we consider refactoring this code to use constants instead of explicit image names?

Comment (zionyx) [reply Test-Source/Invoke-DockerPull.Tests.ps1]: Yes we can. However these image names are made up for the tests. :sweat_smile:

File: Test-Source/Invoke-DockerPull.Tests.ps1 (hunk 1 above)
Comment (zionyx) [reply Test-Source/Invoke-DockerPull.Tests.ps1]: Looks like a quick refactoring at global scale. I'll see what i can do this week.

Comment (zionyx) [reply Test-Source/Invoke-DockerPull.Tests.ps1]: Done.

**Comment by zionyx** [comment]
Rebased to master to resolve conflicts.

**Pull request merged**